packages/cli-kit/src/public/node/ruby.ts

```diff
@@ -136,31 +138,24 @@ async function installThemeCheckCLIDependencies(stdout: Writable) {
  * Install RubyCLI and its dependencies
  * Shows a loading spinner if it's the first time installing dependencies
  * or if we are installing a new version of RubyCLI.
+ *
+ * @param stdout - The Writable stream on which to write the standard output.
  */
-async function installCLIDependencies() {
+async function installCLIDependencies(stdout: Writable) {
   const exists = await file.fileExists(shopifyCLIDirectory())
-  const tasks = [
-    {
-      title: 'Installing theme dependencies',
-      task: async () => {
-        const usingLocalCLI2 = Boolean(process.env.SHOPIFY_CLI_2_0_DIRECTORY)
-        await validateRubyEnv()
-        if (usingLocalCLI2) {
-          await bundleInstallLocalShopifyCLI()
-        } else {
-          await createShopifyCLIWorkingDirectory()
-          await createShopifyCLIGemfile()
-          await bundleInstallShopifyCLI()
-        }
-      },
-    },
-  ]
 
-  if (exists) {
-    await tasks[0]!.task()
+  if (!exists) stdout.write('Installing theme dependencies...')
+  const usingLocalCLI2 = isTruthy(process.env.SHOPIFY_CLI_2_0_DIRECTORY)
```

Comment:
`isTruthy` `return`s `['1', 'true', 'TRUE', 'yes', 'YES'].includes(variable)`
but `SHOPIFY_CLI_2_0_DIRECTORY` is e.g.`"/Users/poitrin/src/github.com/Shopify/shopify-cli"`
Therefore, this can't work :-(